LiveSplit.AutoSplitters.xml

```diff
@@ -20707,4 +20707,15 @@
         <Description>Auto Splitter for Obama (by CodeNameMeteor)</Description>
         <Website>https://github.com/CodeNameMeteor/Autosplitters/blob/main/Obama/ObamaAny.asl</Website>
     </AutoSplitter>
+    <Autospliter>
+    <Games>
+        <Game>Assassin's Creed Syndicate/Jack-the-Ripper<Game>
+    </Games>
+    <URLs>
+        <URL>https://raw.githubusercontent.com/TpRedNinja/Assassins-Creed-Syndicate/main/ACS.asl</URL>
+    </Urls>
+    <Type>Script</Type>
+    <Description>Auto Splitter for Assassins Creed Syndicate (by TpRedNinja)</Description>
+    <Website>https://github.com/TpRedNinja/Assassins-Creed-Syndicate/blob/main/ACS.asl</Website>
+    </Autospliter>
```

Comment:
```suggestion
    <Autosplitter>
        <Games>
            <Game>Assassin's Creed Syndicate<Game>
        </Games>
        <URLs>
            <URL>https://raw.githubusercontent.com/TpRedNinja/Assassins-Creed-Syndicate/main/ACS.asl</URL>
        </Urls>
        <Type>Script</Type>
        <Description>Auto Splitter for Assassins Creed Syndicate (by TpRedNinja)</Description>
        <Website>https://github.com/TpRedNinja/Assassins-Creed-Syndicate/blob/main/ACS.asl</Website>
    </Autosplitter>
```

Reply:
so you want me to remove the /jack the ripper part? and thats all?

Reply:
You can simply click "Commit suggestion".

Reply:
oh well i just edit it i just moved the jack the ripper part to the description as currently only works for that working on it working for base game.